src/main/python/ttconv/imsc/attributes.py

```diff
@@ -297,6 +299,26 @@ class TemporalAttributeParsingContext:
   frame_rate: Fraction = Fraction(30, 1)
   tick_rate: int = 1
 
+@dataclass
+class TemporalAttributeWritingContext:
+  frame_rate: Fraction = Fraction(30, 1)
```

Comment:
Wouldn't it be simplier to initialize this `frame_rate` attribute into a `__init__` with a default `Fraction` value?

Reply:
@dataclass automatically adds the __init__ 

https://docs.python.org/3/library/dataclasses.html

I'm not sure if adding our own __init__ would make this more clear or not.

@palemieux Do you have any thoughts on this?